src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -101,6 +101,22 @@ export const BaseComment = ({
     const [isEditing, setIsEditing] = React.useState<boolean>(false);
     const [isInputOpen, setIsInputOpen] = React.useState<boolean>(false);
 
+    const prevStatus = React.useRef<?FeedItemStatus>(status);
+
+    React.useEffect(() => {
+        if (
```

Comment:
If statement is a little long, maybe we can store the statements in 2 boolean variables like `newStatusIsResolved` and `repliesIsExpanded` to improve readability?